### PR TITLE
docs: add content to getting started page

### PR DESCRIPTION
**Pull request by HalilFocic**
## PR Checklist

- [x] Addresses an existing open issue:  #345
- [ ] You have discussed this issue with the maintainers of `maxGraph`, and you are assigned to the issue.
- [x] The scope of the PR is sufficiently narrow to be examined in a single session. A PR covering several issues must be split into separate PRs. Do not create a large PR, otherwise it cannot be reviewed and you will be asked to split it later or the PR will be closed.
- [ ] I have added tests to prove my fix is effective or my feature works. This can be done in the form of automatic tests in `packages/core/_tests_` or a new or altered Storybook story in `packages/html/stories` (an existing story may also demonstrate the change).
- [ ] I have provided screenshot/videos to demonstrate the change. If no releavant, explain why.
- [x] I have added or edited necessary documentation, ~~or no docs changes are needed.~~
- [x] The PR title follows the ["Conventional Commits"](https://www.conventionalcommits.org/en/v1.0.0/) guidelines.

## Overview

Since we didn't have any getting started page on our docs, I took some time to add basic getting started guide which includes the following:
- Installing via npm, pnpm or yarn
- importing the module and displaying first graph
- Explaining core graph parts (vertex, edge and group).
- How to handle events (click and cells moved event)
- Basic vertex styling

## Notes

This probably is not the final version of the Getting started docs but it is sure a better version than what we have now.

## Summary by CodeRabbit

- **New Features**
	- Enhanced "Getting Started" documentation with a work-in-progress warning and updated licensing information.
	- Added sections for "Installation" and "Basic Usage" with detailed examples and code snippets.
	- Introduced subsections on "Basic Operations" covering adding elements and event handling.
	- Included a "Need Help?" section for user support resources.
	- Expanded installation instructions in the README to include npm, yarn, and pnpm commands.
	- Updated the "Getting Started" section with a detailed example of displaying a rectangle connected to an orange circle.

- **Bug Fixes**
	- Corrected the link to the GitHub releases page for the `maxGraph` package.

### CODE REVIEW

File: packages/website/docs/manual/assets/getting-started/first-graph.gif
Comment (tbouffard) [inline packages/website/docs/manual/assets/getting-started/first-graph.gif:1]: **suggestion:** This file seems duplicated with the file used in the README.
Wouldn't be possible to 
- remove this file
- move the gif file used in the README here
- make the README use the git file of the documentation

**Review by tbouffard — COMMENTED** [review]
Thanks for this PR, this looks great.
I will try to have a look at it this week (I have currently limited time to work on maxGraph).


In the meantime, I will answer to your question about the documentation roadmap in #595. IMHO, it is important we agree on a plan for the documentation. For example, what we put in the README (to limit the duplication with the documentation  to reduce maintenance needs), what we keep from the former mxGraph documenation.

File: packages/website/docs/manual/getting-started.md [hunk 1]
```diff
@@ -32,6 +32,181 @@ Copyright (c) JGraph Ltd 2006-2020
...
+- you are building an application that includes the maxGraph dependency, and it has been installed as explained above.
+- your application uses a build tool or a bundler for its packaging. Direct usage of maxGraph in a web page is not supported.
+- your application includes a page that defines an element with the id graph-container.
+you want to use TypeScript, adapt it if you want to use JavaScript (mainly, remove references to the 'type' syntax).
+
+
+```javascript
+import {type CellStyle, Graph, InternalEvent} from '@maxgraph/core';
+// Make sure you have element with id 'graph-container' in your HTML
+const container = <HTMLElement>document.getElementById('graph-container');
```
[the remark this answers is not in the document]
Comment (tbouffard) [reply packages/website/docs/manual/getting-started.md]: @coderabbitai We want to keep the code simple here as this is an example. We can safely assume that the 'graph-container' exists and is an `HTMLElement`

File: packages/website/docs/manual/assets/getting-started/first-graph.gif
Comment (tbouffard) [reply packages/website/docs/manual/assets/getting-started/first-graph.gif:1]: Done in 59e6285  and e57e860

**Review by tbouffard — APPROVED** [review]
LGTM, thanks for this contribution @HalilFocic 

I took the liberty of making a few changes directly in the branch of this PR:
- use the same gif in the README and documentation
- mark the parts of the README that are duplicated in the documentation for easier maintenance
- use tabs in the installation paragraph to shorten content
- a few styling adjustments

The content of this getting started section can be repeated later, once the mxGraph tutorial has been integrated into the documentation.

I'm going to leave this PR open for a few days to allow you to give feedback if the changes I've applied cause any problems.

**Comment by HalilFocic** [comment]
> I'm going to leave this PR open for a few days to allow you to give

Thanks. I will check it tomorrow first thing in the morning. We can chat more about docs parts in the dicsussion I started. I am willing to translate the existing mxGraph to maxGraph docs but I think we should structure it differently so it's easier for newcommers to adopt max graph.

**Comment by tbouffard** [comment]
> > I'm going to leave this PR open for a few days to allow you to give
> 
> Thanks. I will check it tomorrow first thing in the morning. We can chat more about docs parts in the dicsussion I started. I am willing to translate the existing mxGraph to maxGraph docs but I think we should structure it differently so it's easier for newcommers to adopt max graph.

I'm thinking exactly the same thing and I also wanted to share some thoughts on the subject and how we might move forward on this theme.
Unfortunately, I don't have much time at the moment and haven't found any to share everything 😸 . Hopefully I'll be able to do it this week, otherwise it'll probably be early 2025.

**Comment by HalilFocic** [comment]
> I'm thinking exactly the same thing and I also wanted to share some thoughts on the subject and how we might move forward on this theme.
> Unfortunately, I don't have much time at the moment and haven't found any to share everything 😸 . Hopefully I'll be able to do it this week, otherwise it'll probably be early 2025.

LGTM.

Sorry, got super busy since it's end of the year. We can maybe try make some roadmap and tasks in the January. I can take the toll of writing the docs but I would like for us to sync and be on the same page about which direction are we going. If there is any non docs related bugs/features I could take a lot until end of the year, please feel free to link them in reply or assign me to them.

**Review by tbouffard — APPROVED** [review]
LGTM

**Pull request merged**